example-gateway-givewp.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Plugin Name: Give - Example Gateway
+ * Plugin Name: Example Gateway for GiveWP
```

Comment:
@Benunc `Give - {Plugin Name}` is the standard for give add-ons (for consistency). Are you intending to deviate from that for this?

Reply:
Yes, because this is an example for how third parties should make add-ons, which is diferent.